src/Sulu/Bundle/MediaBundle/Admin/MediaAdmin.php

```diff
@@ -82,6 +82,13 @@ function(Localization $localization) {
             (new Route('sulu_media.overview', '/collections/:locale/:id?', 'sulu_media.overview'))
                 ->addOption('locales', $mediaLocales)
                 ->addAttributeDefault('locale', $mediaLocales[0]),
+            (new Route('sulu_media.detail', '/media/:locale/:id', 'sulu_admin.resource_tabs'))
+                ->addOption('resourceKey', 'media')
+                ->addOption('locales', $mediaLocales),
+            (new Route('sulu_media.detail.detail', '/details', 'sulu_media.detail'))
```

Comment:
That sounds strange 😕 I would go for `sulu_media.form.detail` (as e.g. in the SnippetBundle) and rename the ResourceTab route to `sulu_media.form`.